tests/verify_basic_upgrade.erl

```diff
@@ -22,15 +22,41 @@
 -export([confirm/0]).
 -include_lib("eunit/include/eunit.hrl").
 
+-define(NUM_KEYS, 20000).
+
+-define(CONFIG(RingSize, NVal), 
+    [
+        {riak_core,
+            [
+            {ring_creation_size, RingSize},
+            {default_bucket_props,
+                [
+                    {n_val, NVal},
+                    {allow_mult, true},
+                    {dvv_enabled, true}
+                ]}
+            ]
+        },
+        {leveled,
+            [
+                {journal_objectcount, 2000} 
+                % setting  low object count ensures we test rolled journal
+                % files, not just active ones
+            ]}
+        ]).
+
 confirm() ->
+    
     TestMetaData = riak_test_runner:metadata(),
     OldVsn = proplists:get_value(upgrade_version, TestMetaData, previous),
 
-    Nodes = [Node1|_] = rt:build_cluster([OldVsn, OldVsn, OldVsn, OldVsn]),
+    [Nodes] = 
```

Comment:
This looks different to the eye.
Is there an extra list around result of `rebuild_cluster` now..?

Reply:
Ok, it's `rebuild_clusters` with an `s`.